action.yml

```diff
@@ -16,10 +16,12 @@ inputs:
     description: Use tool cache to speed up installation
     required: false
     default: false
-  use-cache:
```

Comment:
This key is removed. (Breaking change?)

Reply:
Yeah, that would be a breaking change, but considering that these changes will be published as a next minor (or maybe even major) version, it is okay to drop that.